bin/build-and-push.sh

```diff
@@ -14,15 +14,14 @@ _appEnvironment() {
   fi
 }
 
-
 cwd=$(dirname $(realpath $0))
 
 imageName=${1:?"app name is required as first argument"}
 gitRepo=${2:?"git repository URL is required as second argument"};
 gitCommit=${4:?"git commit SHA is required as fourth argument"};
 gitBranch=${3:?"git branch is required as third argument"};
 
-gitShortCommit=$(echo $CIRCLE_SHA1 | cut -c -7)
+gitShortCommit=$(echo $gitCommit | cut -c -7)
```

Comment:
What does the -7 flag? 🤔 
Are you selecting `-7` with `-c`? 🤔

Reply:
```shell-session
❯ echo 1234567890 | cut -c -7
1234567
```

Reply:
yep